sdk/extensions/Azure.Extensions.AspNetCore.DataProtection.Keys/src/ConfigureKeyManagementKeyVaultEncryptorClientOptions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.DataProtection.KeyManagement;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+#pragma warning disable AZC0001 //
```

Comment:
nit:  Can we please leave a short description of what is being disabled?

Reply:
Wasn't necessary - Builds without any warnings even without. Have removed